style.css

```diff
@@ -284,6 +284,81 @@ header a {
   transform: scale(1.1);
 }
 
+.contact-section {
+  display: flex;
+  flex-direction: column;
+  align-items: center;
+  background-color: #6070ff;
+  border-top-left-radius: 80px;
+  width: 100%;
+  background-image: url(/images/contact-form-m-bg.svg);
+  background-size: contain;
+  background-position: top right;
+  background-repeat: no-repeat;
+  padding: 50px 24px;
+}
+
+.contact-section h2 {
+  color: #fff;
+  width: 90%;
+  text-align: center;
+  padding-top: 25px;
+  font-weight: 700px;
+  font-size: 40px;
+}
+
+.contact-section p {
+  color: #ebebff;
+  font-weight: 400;
+  font-size: 20px;
+  line-height: 28px;
+  text-align: center;
+  margin-top: -10px;
+  padding: 30px;
+}
```

Comment:
Please consider reviewing your fonts in accordance with the template fonts. The font family used in the Figma template is `    font-family: 'Poppins';` They might not be available generally so you can download or use the CDN link of Google fonts for them to use.